src/assets/locales/hi.js

```diff
@@ -557,6 +557,8 @@ export default {
       num_questions_answered: {
         title: "उत्तर किये गए प्रश्न",
       },
+      survey_warning:
+        "*किसी भी मीट्रिक की गणना में सर्वे मोड के प्रश्नों पर विचार नहीं किया जाता है",
```

Comment:
don't rely completely on google translate - you often have to read the sentence out loud again, use different words, change the ordering a bit to get the right, meaningful translation.

```suggestion
        "सर्वेक्षण मोड के प्रश्नों को  किसी भी आंकड़े की गणना में शामिल नहीं हैं",
```